mlflow/R/mlflow/tests/testthat/test-model.R

```diff
@@ -9,6 +9,8 @@ teardown({
 })
 
 test_that("mlflow can save model function", {
+  skip_on_windows("ValueError: close_fds is not supported on Windows platforms if you redirect stdin/stdout/stderr")
```

Comment:
For this to work we just need to switch to a version of Python higher than 3.6 that is built with https://bugs.python.org/file46175/windows-subprocess-close-fds.patch

If I remember correctly, Python 3.8 would do.